ciao-launcher/qemu.go

```diff
@@ -358,7 +358,8 @@ func launchQemuWithNC(params []string, fds []*os.File, ipAddress string) (int, e
 		ncString := "socket,port=%d,host=%s,server,id=gnc0,server,nowait"
 		params[len(params)-1] = fmt.Sprintf(ncString, port, ipAddress)
 		var errStr string
-		errStr, err = qemu.LaunchQemu(context.Background(), params, fds, qmpGlogLogger{})
+
+		errStr, err = qemu.LaunchCustomQemu(context.Background(), "", params, fds, qmpGlogLogger{})
```

Comment:
@markdryan so the device creation order (and hence insertion into the slice) should match the append order.